### ISSUE TITLE
Use the same placeholder multiple times

### ISSUE THREAD

**Issue by NicolasRoehm** [Gregwar/Formidable#57]
I cannot use a placeholder more than once :

**HTML**
```html
<form>
    <h1>Hello {{ name }}</h1>
    <p>{{ name }}</p>
    <!-- ... -->
</form>
```
**PHP**
```php
$form = new Gregwar\Formidable\Form('forms/myform.html');
$form->setPlaceholder('name', 'Bob');
echo $form;
```
**RESULT**
```html
<form method="post">
    <h1>Hello </h1>
    <p>Bob</p>
    <!-- ... -->
</form>
```
Is it possible to allow such behavior, please ?
Thank you for sharing this library, it's very useful and working well! :)

**Comment by Gregwar** [comment]
Hey, this should be fixed by 927e20e 
Thanks for reporting, please confirm and close issue if it's ok for you

**Comment by NicolasRoehm** [comment]
Wow thanks for the quick fix! It works like a charm :)
